var/spack/repos/builtin/packages/jq/package.py

```diff
@@ -34,3 +34,4 @@ class Jq(AutotoolsPackage):
     version('1.5', 'c8070bd6ec275404f77db3d2e568c9a3')
 
     depends_on('oniguruma')
+    depends_on('bison@3.0:')
```

Comment:
The error message has been blown away, I can't recreate it exactly.  Something to the effect of providing a bison at version 3.0 or newer or disabling maintainer mode.

Should it just be a build dep?